ks/vojtux_common.ks

```diff
@@ -182,14 +184,6 @@ rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-rpmfusion-free-fedora-*-primary
 echo "Importing RPM Fusion keys"
 rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-rpmfusion-nonfree-fedora-*-primary
 
-# import Vojtux-apps key
```

Comment:
Hello, something probably went wrong during the rebase... we don't want this part to be removed, please add it back or remove the commit which removes it.

Reply:
Sorry about that. Added it back.